libraries/chain/chain_controller.cpp

```diff
@@ -1273,6 +1290,12 @@ chain_controller::~chain_controller() {
 void chain_controller::replay() {
    ilog("Replaying blockchain");
    auto start = fc::time_point::now();
+
+   auto on_exit = fc::make_scoped_exit([this](){
```

Comment:
Limit lambda capturing to just one needed parameter.

Reply:
Done.